README.md

```diff
@@ -18,19 +18,72 @@ We are going to track some new features/additions here so that it is quick and e
 - we've renamed the distributed .env files to ```switchboard.env.dist``` and ```frontend.env.dist```. This ensures that your local
   configuration doesn't get blown away when you pull changes from the repo. (We still use ```switchboard.env``` and ```frontend.env```
   for the config, it just means that new clones of the repo require the users to copy / rename the dist files)
-
 - we have added an extra `switchboard.env` called `CANARY_IPINFO_API_KEY`. This allows you to use your ipinfo.io api key if you
   want to (keep in mind ipinfo.io does have a free tier of up to 1000 requests a day).
-
 - we now have slack support. When you supply a webhook, you simply supply your slack webhook url. (Thanks to @shortstack).
-
 - we have added a new environment variable to `frontend.env` called `CANARY_AWSID_URL` which allows you to specify a private or
   different url for the AWS ID token. This means you can easily change between accounts. (2018-10-17)
 
+
+
+## Domain name / DNS configuration
+
+You register a domain name using a "registrar". This "registrar" has a communication channel with the "registry" which is the entity managing the TLD (`.com` for example).
+
+The registry operate some DNS servers that list domain names available into it's TLD, with the corresponding "next" DNS servers in the chain to send requests to for further records inside this domain zone.
+
+You configure in admin panel of you registrar, what DNS servers you want the registrar to "push" into the DNS servers managed by the registry.
+
+
+
+- Registry DNS level
+
+This information is defined into the Registrar admin panel (then pushed to Registry DNS servers)
+
+```
+example.com.    NS    ns1.my.dns.servers.
+example.com .   NS    ns2.my.dns.servers.
+```
+
+
+
+- Domain DNS level
+
+On `nsX.my.dns.servers.`  DNS servers, configure the following example of zone content
+
+```
+ns.example.com.       A     1.2.3.4
+example.com.          NS    ns.example.com.
+example.com.          A     1.2.3.4
+nx.example.com.       NS    ns.example.com.
+```
+
+You must set your CanaryToken server as "NS" for your domain name. NS record can't be an IP address, so you must also define a name. Here we  decided to name our "NS" server with the not-too-suspicious name "ns" just because "canarytoken" may looks far too suspicious :)
```

Comment:
You must set your Canarytokens server as the NameServer (NS) for your domain name (`example.com`). 

We have done this by telling the DNS resolvers that to resolve anything relating to `example.com`, please contact `ns.example.com`. Since `ns.example.com` is a subdomain of `example.com`, we need a glue record (A) to specify the address to query. By adding these records you have made your Canarytokens server at `1.2.3.4` the authoritative server for `example.com`.

Reply:
We have also added the `nx.example.com`. We use this in our Canarytokens config (`CANARY_NXDOMAINS`) which is used for the pdf Canarytoken. You can read about that a little later.

Reply:
- Sounds good to me.
Playing a bit more on the last days, it looks the embeded DNS server of the canarytoken server always reply to NS queries with harcoded "ns1" like  `ns1.example.com`  ?
So may be better (even if not mandatory) to suggest people to create "ns1" instead of "ns"  by providing "ns1" as example ?

- Ok for now about "nx" part.. even if I have not yet all understood about it :) 
I mean, I'm still not sure how this is different to a regular dns token.. etc.. I did few tests and still confused.. :)
and still not sure why this is called "nx" / "CANARY_NXDOMAINS" as it looks it tokens like "token.nx.example.com" does resolve fine without a "NXDOMAIN" DNS status ..   Anyway, I think outside the scope of this MR for now, I will continue to test more and possibly open a new "issue" on this to get more informations from you ;)